examples/js/loaders/ColladaLoader.js

```diff
@@ -3190,6 +3190,11 @@ THREE.ColladaLoader.prototype = {
 			for ( var i = 0, l = keys.length; i < l; i ++ ) {
 
 				var id = instanceMaterials[ keys[ i ] ];
+
+				if( id === undefined ) { // Fallback if no material matches
+					id = instanceMaterials[Object.keys(instanceMaterials)[0]] // take simply the first material 
+				}
```

Comment:
I would write it like this:
```javascript
if ( id === undefined ) {

   console.warn( 'THREE.ColladaLoader: Material with key %s not found. Use fallback material.', keys[ i ] );
   id = instanceMaterials[ Object.keys( instanceMaterials )[ 0 ] ];

}
```
It's important to me to add a warning, see https://github.com/mrdoob/three.js/pull/12866#discussion_r156474476. With the log message, your additional comments are not necessary.

Reply:
Good idea. Done :)